codalab/lib/bundle_cli.py

```diff
@@ -1730,6 +1732,29 @@ def wrap(string):
                 self.print_target_info(client, (bundle_uuid, item['name']), decorate=True)
 
     @Commands.command(
+        'mount',
+        help=[
+            'Mount the contents of a bundle at a read-only mountpoint.',
+        ],
+        arguments=(
+            Commands.Argument('target_spec', help=TARGET_SPEC_FORMAT, completer=TargetsCompleter),
+            Commands.Argument('--mountpoint', help='Empty directory path to set up as the mountpoint for FUSE.'),
```

Comment:
I'm a bit confused what the concern is here.. I thought the idea was just that
```
cl mount data
```
would just be equivalent to
```
cl mount data data
```